src/Abstractions/RetryPolicy.cs

```diff
@@ -132,6 +135,8 @@ public class RetryPolicy
     /// <value>
     /// Defaults delegate that always returns true (i.e., all exceptions are retried).
     /// </value>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    [Obsolete("This functionality is not implemented. Will be removed in the future. Use TaskOptions.FromRetryHandler or HandleFailure instead.")]
```

Comment:
Oh, you didn't need to implement `Obsolete` on this method. I just wanted the `EditorBrowsable` attribute.